server/src/test/java/org/candlepin/controller/refresher/visitors/ContentNodeVisitorTest.java

```diff
@@ -332,8 +277,10 @@ public void testProcessNodeForUnchangedEntity(String key, Object base, Object up
         Owner owner = TestUtil.createOwner();
         String id = TestUtil.randomString("test_id");
 
-        Content existing = this.createPopulatedEntity(id, key, base);
-        ContentInfo imported = this.createPopulatedEntity(id, key, base);
+        Content existing = this.createPopulatedExistingEntity(id, key, base);
+        ContentInfo imported = this.createPopulatedImportedEntity(id, key, base);
+
+        MergeValidator<ContentInfo> validator = this.validators.get(key);
```

Comment:
It doesn't look like this one line is needed (the validator isn't used thereafter)

Reply:
Fixed